src/plugins/news/services/ArticleSubscriber.js

```diff
@@ -98,11 +98,29 @@ export default class ArticleSubscriber extends EventSubscriber {
   onValidateForm(formEvent) {
     const data = formEvent.getData();
     const node = formEvent.getMessage();
+    const redux = formEvent.getRedux();
+    const { title } = data;
+    const { TITLE_LENGTH_LIMIT } = formRules;
+    let meta = {};
 
-    if (!data.title) {
+    if (redux) {
+      meta = getFormMeta(formEvent.getName())(redux.getState());
+    }
+
+    if (!title) {
       formEvent.addError('title', 'title is required');
     }
 
+    if (title && title.length >= TITLE_LENGTH_LIMIT + 15) {
+      formEvent.addError('title', `recommendation: keep title less than ${TITLE_LENGTH_LIMIT} characters to avoid title extending too long in search results. (${title.length}/${TITLE_LENGTH_LIMIT})`);
```

Comment:
@bhsiao55 - this is a form validation error, and it prevents the form submission, which is not the desired behavior. Please move this logic to `onWarnForm`